api/v1beta1/verticadb_types.go

```diff
@@ -251,6 +251,12 @@ type VerticaDBSpec struct {
 	SSHSecret string `json:"sshSecret,omitempty"`
 }
 
+// Is used instead of corev1.LocalObjectReference and behaves the same.
```

Comment:
minor: comments like these should start with the name of the type, which is LocalObjectReference.  It is a golang thing.  Same comment for Affinity.